llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1381,16 +1384,73 @@ void VPWidenCastRecipe::execute(VPTransformState &State) {
   }
 }
 
+static bool isCastInstruction(unsigned Opcode) {
```

Comment:
Could it replaced by isCast?https://llvm.org/doxygen/classllvm_1_1Instruction.html#a27997849d8982bf226891024fd68daee